WalletWasabi.Gui/WalletWasabi.Gui.csproj

```diff
@@ -56,11 +56,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Avalonia.Desktop" Version="0.9.5" />
-    <PackageReference Include="Avalonia.Native" Version="0.9.6" />
-    <PackageReference Include="AvalonStudio.Shell" Version="0.9.5" />
-    <PackageReference Include="Dock.Avalonia.Themes.Default" Version="0.9.5" />
-    <PackageReference Include="Dock.Avalonia.Themes.Metro" Version="0.9.5" />
+    <PackageReference Include="Avalonia.Desktop" Version="0.9.6" />    
+    <PackageReference Include="AvalonStudio.Shell" Version="0.9.6" />
```

Comment:
Safe to remove Native?

Reply:
Yes, i had added it as a workaround (master have native (osx native lib) at 0.9.6.

But it can be remove now because we can now target shell and dock at 0.9.6 and they too target avalonia 0.9.6